src/components/ui/TripCard/AccordionDetails/TripCardDetails.tsx

```diff
@@ -0,0 +1,80 @@
+import { useTranslation } from 'react-i18next';
+import DeleteForeverIcon from '@mui/icons-material/DeleteForever';
+
+export const TripCardDetails = () => {
+    const [selectedSeats, setSelectedSeats] = useState<number[]>([]);
+
+    const updateSelectedSeats = (newSeat: number) => {
+        if (selectedSeats.includes(newSeat)) {
+            setSelectedSeats((prev) =>
+                prev.filter((selectedSeat) => selectedSeat != newSeat)
+            );
+        } else setSelectedSeats((prev) => [...prev, newSeat]);
+    };
+    const clearSelectedSeats = () => {
+        setSelectedSeats([]);
+    };
+
+    const { t } = useTranslation('tripDetails');
+
+    return (
+        <AccordionDetails>
+            <Stack direction={'column'} p={3} pt={3}>
+                <Stack direction={'column'} spacing={2}>
+                    <Stack
+                        direction={'row'}
+                        justifyContent={'space-between'}
+                        alignItems={'center'}
+                        spacing={3}
+                    >
+                        <SeatLegend />
+                        {selectedSeats.length <= 0 ? (
+                            <StyledAlert variant="filled" severity="info">
+                                {t('info')}
+                            </StyledAlert>
+                        ) : (
+                            <StyledAlert
+                                action={
+                                    <Tooltip title="Delete selection" arrow>
+                                        <IconButton
+                                            aria-label="delete"
+                                            color="inherit"
+                                            size="small"
+                                            onClick={() => {
+                                                clearSelectedSeats();
+                                            }}
+                                        >
+                                            <DeleteForeverIcon fontSize="inherit" />
+                                        </IconButton>
+                                    </Tooltip>
+                                }
+                                sx={{ mb: 2 }}
+                            >
+                                {t('selectAlertTitle')}
+                                {selectedSeats.length > 1 ? 's' : ''}{' '}
+                                {selectedSeats.join(', ')} {t('selectAlertMsg')}
+                            </StyledAlert>
+                        )}
+                    </Stack>
+                    <SeatLayout
+                        layoutName={layoutNames.volvo25}
+                        seats={seats}
+                        selectedSeats={selectedSeats}
+                        updateSelectedSeats={updateSelectedSeats}
+                    />
+                </Stack>
+
+                <Checkout noOfSeats={selectedSeats.length} farePerSeat={1200} />
```

Comment:
Why don't we define a constant instead of a hardcoded value?

Reply:
Resolved in 308e5f2b802bf6010c03123b1913c1c7d500ee91